### PR TITLE
feat: cleanup several GA feature flags which should be removed in 1.17

### PR DESCRIPTION
**Pull request by draveness**
/kind cleanup
/priority backlog

**What this PR does / why we need it**:

Remove several GA features flag in 1.17

+ EnableAggregatedDiscoveryTimeout @deads2k 
+ PersistentLocalVolumes @msau42 
+ deprecatedGCERegionalPersistentDisk @verult 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
The deprecated feature gates `GCERegionalPersistentDisk`, `EnableAggregatedDiscoveryTimeout` and `PersistentLocalVolumes` are now unconditionally enabled and can no longer be specified in component invocations.
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by tengqm** [comment]
Thanks.

**Comment by deads2k** [comment]
> * PersistentLocalVolumes @msau42

@sjenning @gnufied please confirm.

File: pkg/features/kube_features.go [hunk 1]
```diff
@@ -496,13 +482,6 @@ const (
 	//
 	// Enables the startupProbe in kubelet worker.
 	StartupProbe featuregate.Feature = "StartupProbe"
-
-	// owner @deads2k
-	// deprecated: v1.16
-	//
-	// Enable the aggregated discovery timeout to ensure client responsiveness. Note this feature is present
-	// only for backward compatibility, it will be removed in the 1.17 release.
-	EnableAggregatedDiscoveryTimeout featuregate.Feature = "EnableAggregatedDiscoveryTimeout"
```
Comment (deads2k) [inline pkg/features/kube_features.go]: this one lgtm

**Comment by gnufied** [comment]
Local storage change looks good.

**Comment by msau42** [comment]
/approve
PersistentLocalVolumes and deprecatedGCERegionalPersistentDisk feature gate removal

**Comment by draveness** [comment]
@deads2k please take another look since the other two feature gates removal have been approved by @msau42, thanks

**Pull request merged**